### ISSUE TITLE
Unable to connect cypress-intellij-reporter

### ISSUE THREAD

**Issue by visitek** [mbolotov/intellij-cypress#25]
I have reporter in cypress HomeDir node_modules.
I have also reporter in project node_modules.

I've tried duplicating node_modules with reporter rependencies also Into almost every folder.

When I've checked source code, it should work definitely, but it doesn't.

I'm not about to subscribe for all in my team, because I was unable to try it.

![image](https://user-images.githubusercontent.com/1741839/97203277-e21c7500-17b4-11eb-8293-51a6e93ba7a2.png)

**Comment by visitek** [comment]
![image](https://user-images.githubusercontent.com/1741839/97203671-71298d00-17b5-11eb-96b1-8d873889678d.png)

**Comment by visitek** [comment]
![image](https://user-images.githubusercontent.com/1741839/97203898-b948af80-17b5-11eb-803c-4fc1380d72fe.png)

**Comment by mbolotov** [comment]
Hi @visitek 
Do you have your spec file outside of the Cypress project base one?

**Comment by visitek** [comment]
Sure, it is transpiled by the typescript to dist

**Comment by mbolotov** [comment]
Okay, this is a current limitation of the plugin. It can't handle such a case. 
I will look if I can fix this.

**Comment by visitek** [comment]
So my trial is expiring as well and we've not tested because we've got stucked

**Comment by mbolotov** [comment]
Could you configure your project to have the spec inside the base directory?

**Comment by visitek** [comment]
So we need to set the tsconfig to push the output into directory where are sources? This does not seem as clean solution at all.

**Comment by mbolotov** [comment]
This is all I can offer you until I fix this issue and release a new version of plugin

**Comment by visitek** [comment]
Sure, thank you.. we will try it temporarily like that.

**Comment by mbolotov** [comment]
hi @visitek!
Could you check if this issue was fixed in the 1.6.2 version?

**Comment by visitek** [comment]
looks fixed. Thank you.
One more thing is not clear to me. How can I set the runner script?
Due to fact the tests are in typescript and we use Angular, the runner should be called with `ng e2e`.

Currently, we must use a workaround with tsc watch. But the tests do not accept .ts files, only .js files.

Do you have any idea? Maybe I've missed something.

**Comment by visitek** [comment]
One more thing we still must fix manually.
Path to the base node_modules does not work for mocha.. we must set the path manually.

<img width="722" alt="fixReporter" src="https://user-images.githubusercontent.com/1741839/98658811-bfa56280-2343-11eb-8eac-619231b98010.png">

**Comment by mbolotov** [comment]
> Do you have any idea? Maybe I've missed something.

No, there is currently no way to use a script to run the tests. 
How do you call the runner? Could you provide the full command?

**Comment by visitek** [comment]
When we use JEST for units, there is a just bin path when we can provide path to `node_modules/@angular/cli` and this automatically uses `ng test`. It would expect similar behavior. But this does not matter, it works. Bigger problem is, that it does not support `.ts` files. The plugin helps, but not as much as it could.

It uses global cypress installed in my home directory into `mainFile`.
I think that the full configuration would be more clear than the magic, but it works if someone wants. I think that mocha installed globally would work finally.

**Comment by mbolotov** [comment]
I mean how do you start Cypress from the command line, with no help from the plugin. Could you provide the command?

**Comment by visitek** [comment]
just `ng e2e` alternativelly with the cypress parameters `ng e2e --watch`
